example/gluon/word_language_model/train.py

```diff
@@ -58,6 +58,14 @@
                           takes `2bit` or `none` for now.')
 parser.add_argument('--gcthreshold', type=float, default=0.5,
                     help='threshold for 2bit gradient compression')
+parser.add_argument('--hybridize', action='store_true',
+                    help='whether to hybridize in mxnet>=1.3')
+parser.add_argument('--static-alloc', action='store_true',
+                    help='whether to use static-alloc hybridize in mxnet>=1.3')
+parser.add_argument('--static-shape', action='store_true',
+                    help='whether to use static-shape hybridize in mxnet>=1.3')
+parser.add_argument('--export-only', action='store_true',
+                    help='export a symbol graph and exit')
```

Comment:
Please mention the default values for all of these arguments

Reply:
Added defaults as directed.